content/git/concepts/commit/commit.md

```diff
@@ -30,7 +30,7 @@ Before using `git commit`, make sure to have all the neccessary files staged for
 git add <filename>
```

Comment:
```suggestion
$ git add filename
```

Reply:
If we're changing this to filename and keeping the \`\`\`shell designation, should we add a file extension to this example?